lib/models/watcher.js

```diff
@@ -48,7 +48,9 @@ class Watcher extends CoreObject {
 
     if (this.serving) {
       let options = this.options;
-      let baseURL = options.rootURL === '' ? '/' : cleanBaseURL(options.rootURL || options.baseURL);
+      let config = this.options.project.config();
```

Comment:
Should be something like `let config = this.project.config(options.environment)`